tests/ui/test_ui_tools.py

```diff
@@ -415,6 +415,8 @@ def test_init(self, mocker, stream_view):
         ('FOO', ['FOO', 'FOOBAR', 'foo']),
         ('test', ['test here']),
         ('here', ['test here']),
+        ('', ['FOO', 'FOOBAR', 'foo', 'fan',
+            'boo', 'BOO', 'bar', 'test here'])
     ])
     def test_update_streams(self, mocker, stream_view, new_text, expected_log):
         stream_names = [
```

Comment:
@Ar9av I think you just need test if `log.set_focus` was called. No other change in needed I think.